BUILD.bazel

```diff
@@ -1 +1,15 @@
 # TODO(#1532): Rename file to 'BUILD' post-Gradle.
+'''
+Oppia Android binary
```

Comment:
Doc comment probably isn't needed since this is self-documenting.

Reply:
Okay